packages/esm-service-queues-app/src/queue-table/cells/queue-table-actions-cell.component.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, OverflowMenu, OverflowMenuItem } from '@carbon/react';
+import { showModal } from '@openmrs/esm-framework';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { type QueueTableCellComponentProps, type QueueTableColumn } from '../../types';
+
+export function QueueTableActionsCell({ queueEntry }: QueueTableCellComponentProps) {
```

Comment:
Aren't there really 2 different cells here?  One is a cell containing a "Transfer" button.  The other is a cell containing an "Actions" menu.  I could see different implementations wanting one and not the other.